src/docs/common/snippets/common-graalvm-annotation-processor.adoc

```diff
@@ -0,0 +1,9 @@
+The Micronaut GraalVM Annotation processor visits the annotation and provides the reflection metadata with a https://www.graalvm.org/latest/reference-manual/native-image/dynamic-features/Reflection/#configuration-with-features[GraalVM Feature].
+
+dependency:micronaut-graal[groupId=io.micronaut,scope=annotationProcessor]
+
+:exclude-for-build:maven
```

Comment:
should add maven snippet

Reply:
@alvarosanchez what should we write for Maven Plugin?

Reply:
In the case of Maven, `micronaut-graal` processor is defined in the Parent POM, so users don't need to specify it either

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/1346/commits/b3b70452a5dead3069e86adb53f7e3e21cc4495e